src/main/scala/com/github/andyglow/websocket/WebsocketClient.scala

```diff
@@ -90,7 +90,7 @@ object WebsocketClient {
 
   def apply[T: MessageAdapter](uri: Uri)(receive: PartialFunction[T, Unit]): WebsocketClient[T] = apply(uri, WebsocketHandler(receive))
 
-  case class Builder[T: MessageAdapter](
```

Comment:
please don't make them `final`

Reply:
@andyglow Removed all the finals. Thanks for reviewing!